app/models/user.rb

```diff
@@ -33,7 +33,7 @@
 #
 
 class User < ActiveRecord::Base
-  attr_accessible :email, :full_name, :display_name, :password, :password_confirmation, :disabled
+  attr_accessible :email, :full_name, :display_name, :password, :password_confirmation, :disabled, :role, :profile_attributes
```

Comment:
Just seen this is related to a bug.  Could you explain the bug a bit?  I couldn't see an obvious related open issue.